app/routes/wow.weekly-price-group-delta-recommended.tsx

```
@@ -0,0 +1,84 @@
+                  className="text-blue-500 hover:text-blue-600 font-medium">
+                  Weekly Price Group Delta
+                </a>{' '}
+                page to start your analysis from scratch.
+              </p>
+            </div>
+            <div className="not-prose my-12 grid grid-cols-1 gap-6 sm:grid-cols-2">
+              {recommendedConfigs.map((rec) => (
+                <button
+                  key={rec.name}
+                  className="bg-blue-600 hover:bg-blue-700 text-white font-bold py-4 px-6 rounded shadow text-left"
+                  onClick={() => handleRunAnalysis(rec)}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)